nixos/tests/magnetico.nix

```diff
@@ -21,8 +22,8 @@ import ./make-test.nix ({ pkgs, ...} : {
       startAll;
       $machine->waitForUnit("magneticod");
       $machine->waitForUnit("magneticow");
-      $machine->succeed("${pkgs.curl}/bin/curl -u user:password http://localhost:8080");
-      $machine->succeed("${pkgs.curl}/bin/curl -u user:wrongpwd http://localhost:8080") =~ "Unauthorised." or die;
+      $machine->succeed("${pkgs.curl}/bin/curl -u user:password http://localhost:8081");
```

Comment:
This could be `web.port` variable for consistency.

Reply:
Ok, done.